sfge/src/main/java/com/salesforce/graph/ops/registry/Indexable.java

```diff
@@ -0,0 +1,16 @@
+package com.salesforce.graph.ops.registry;
+
+/**
+ * Represents a type that has an id and can be added to the registry. See {@link Registry} for more
+ * information.
+ * To add to the registry, the implementer needs to have a reference to their registry instance
+ * and invoke the register() method probably from the constructor.
+ */
+public interface Indexable {
```

Comment:
Having `Indexable` and `xRegistry` is confusing to me. I feel like it should be `Indexable` and `xIndex` or `Registrable` and `Registry`, instead of mixing and matching.

Reply:
I considered `Registrable` first but didn't like the sound of it. If you feel strongly, I can go back to `Registrable`.

Reply:
I'd advocate for using `Registrable`.